src/FakeItEasy/Core/TypeCatalogue.cs

```diff
@@ -1,4 +1,4 @@
-namespace FakeItEasy.Core
+﻿namespace FakeItEasy.Core

```

Comment:
> I usually try not to commit this kind of change, but I missed it this time.

Me too.

> I can change it back if you prefer.

I think worrying about this is just going to exhaust us.  And _every_ contribution from a community member is going to have the change. I don't want harass them about this.
Just let it go?